### PR TITLE
docs: updates to README FAQ, technical specification, and fixed links

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

This PR improves the following about the README:

- Adds three questions related to project goal and advancement;
- Fixes `flightsim.to` liveries link (requested by @bouveng);
- Specifies APU model and fixes FMGS release in aircraft technical specification;
- Adds an `A320neo` header to technical specification;
- Adds a link to the version guide on `docs` to an FAQ question;
- Clarifies the implication of GPLv3 licensing when using complied files. **Licensing does not change.**

Discord username (if different from GitHub): someperson#4953

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,9 +64,23 @@ No, it is a completely free aircraft, open-source.
 
 Please read the [known Issues and bug reporting](#known-issues-and-bug-reporting) section.
 
+### Is this going to be "study-level"?
+
+While many in the team dislike the term "study-level" as its use is mostly applicable to certified training devices, we are in fact very dedicated on bringing a high fidelity A320neo to the Microsoft Flight Simulator platform.
+
+### Is X feature from the A320neo going to be implemented?
```
Comment (frankkopp) [inline README.md:71]: Better grammar:
Is feature X from the A320neo going to be implemented?

File: README.md [hunk 2]
```diff
@@ -61,9 +64,23 @@ No, it is a completely free aircraft, open-source.
...
 
+### Is this going to be "study-level"?
+
+While many in the team dislike the term "study-level" as its use is mostly applicable to certified training devices, we are in fact very dedicated on bringing a high fidelity A320neo to the Microsoft Flight Simulator platform.
+
+### Is X feature from the A320neo going to be implemented?
+
+Chances are, yes! While we do not guarantee every single detail of the aircraft will eventually be represented, our goal is to produce an extremely accurate simulation based on technical data and real-world testing. This means you can be almost certain every feature of the aircraft will eventually be simulated to the best of the simulator's ability as long as the technical data backs it up.
+
+### Why has X not been implemented yet?
```
Comment (frankkopp) [inline README.md:75]: Consistency with question above:
Why has feature X not been implemented yet?

**Review by frankkopp — CHANGES_REQUESTED** [review]
I suggest use of "feature X" instead of "X feature" or only "X".
Otherwise LGTM

File: README.md [hunk 3]
```diff
@@ -61,9 +64,23 @@ No, it is a completely free aircraft, open-source.
...
+
 ### Why is my version not the same as what I see others using?
 
-We have two versions: stable and development. The stable version is a 'snapshot' of the development which we regard as stable with the current version of the simulator. The developer build is updated daily and is a constant work in progress and although we test thoroughly each update, minor issues may occur from time to time.
+We have two mainline versions: stable and development. The stable version is a 'snapshot' of the development which we regard as stable with the current version of the simulator. The developer build is updated daily and is a constant work in progress and although we test thoroughly each update, minor issues may occur from time to time.
+
+You can read more about the differences between the versions we offer [on our documentation portal](https://docs.flybywiresim.com/fbw-a32nx/fbw-versions/).
```
Comment (bouveng) [inline README.md:84]:
```suggestion

### Is there a list of features or a changelog available?

We do not keep a list of features implemented, but a lengthy [changelog](https://github.com/flybywiresim/a32nx/blob/master/.github/CHANGELOG.md) of what has been implemented, and their associated [pull requests](https://github.com/flybywiresim/a32nx/pulls) can be [found here](https://github.com/flybywiresim/a32nx/blob/master/.github/CHANGELOG.md).

```

Somewhat hard to locate if you do not know github file structure.

**Review by bouveng — APPROVED** [review]

**Review by bouveng — APPROVED** [review]

**Review by bouveng — APPROVED** [review]

**Pull request merged**